rest-api-spec/src/main/resources/rest-api-spec/test/indices.data_stream/40_supported_apis.yml

```diff
@@ -0,0 +1,37 @@
+---
+setup:
+  - do:
+      indices.create_data_stream:
+        name: logs-foobar
+        body:
+          timestamp_field: "@timestamp"
+
+---
+teardown:
+  - do:
+      indices.delete_data_stream:
+        name: logs-foobar
+
+---
+"Verify get index api":
+  - skip:
+      version: " - 7.99.99"
+      reason: "wait until backported"
+
+  - do:
+      indices.get:
+        index: logs-foobar
+  - is_true: logs-foobar-000001
```

Comment:
Maybe we should also check that at least one setting or similar is included in the output?